docs/core/development/policy/coding-standard.rst

```diff
@@ -76,18 +76,19 @@ Indentation is 4 spaces per indent.
 Tabs are not allowed.
 It is preferred that every block appears on a new line, so that control structure indentation is always visible.
 
-Lines are flowed at 79 columns.
+Lines are flowed at 88 columns per `The Black Code Style <https://github.com/psf/black/blob/master/docs/the_black_code_style.md>`_.
```

Comment:
This is OK.  In future, we may want to remove all mention of number of columns, etc.
from this document, and just say that Twisted follows the style guidelines as followed by Black, and just give a pointer Black's guidelines.  In the end, very few people will read this anyways.

As long as they are told during review time to run `tox -e black-reformat` to make their
PR acceptable, that is probably good enough.